packages/material-ui/src/Input/Input.js

```diff
@@ -70,12 +70,42 @@ export const styles = theme => {
       },
     },
     formControl: {
-      'label + &': {
-        marginTop: 16,
+      '&:not($outlined)': {
+        'label + &': {
+          marginTop: 16,
+        },
       },
     },
     focused: {},
     disabled: {},
+    outlined: {
+      '&$multiline': {
+        display: 'inline-flex',
+        boxSizing: 'border-box',
+
+        borderStyle: 'solid',
```

Comment:
In the spec, multiline is shown without a notch. I believe MCW also reverts to a border because it resizes appropriately as a non-absolutely positioned element. I could not find a way to get a responsive svg in this way, though it would be preferable

Reply:
Odd, I can't see an example of multi-line textfield in the spec, only filled. However MCW haven't implemented filled or outline multiline, only textarea, which is technically something different.